_posts/2021-04-14-cockpit-242.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Cockpit 242
+author: marusak
+date: 2021-04-14
+tags: cockpit linux
+slug: cockpit-242
+category: release
+summary: Cockpit with split out Machines page and website refresh
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 242.
+
+
+## Groundwork for Snowpack support
+
+[Snowpack](https://www.snowpack.dev/) is a web project build system alternative to [webpack](https://webpack.js.org/). Previously, when using it to build a Cockpit page which used an [npm module](https://www.npmjs.com/) containing an `@`, such as [@patternfly](https://www.npmjs.com/package/@patternfly/react-core), the page [could not be loaded](https://github.com/cockpit-project/cockpit/issues/15288). This has been fixed.
```

Comment:
```suggestion
[Snowpack](https://www.snowpack.dev/) is a web project build system alternative to [webpack](https://webpack.js.org/). Previously, when using Snowpack to build Cockpit pages using an [npm module](https://www.npmjs.com/) with an `@`, the page [could not be loaded](https://github.com/cockpit-project/cockpit/issues/15288). This has been fixed. As Cockpit uses [@patternfly](https://www.npmjs.com/package/@patternfly/react-core) in this style, it was an important first step.
```

This is really wordy overall and I lost what "it" was referring to. (I _know_ what is meant, but because there are so many different things it could refer to, and the last thing mentioned is webpack, I decided being specific in this case is best.)

Additionally, I couldn't find a great way to break up the sentence and keep the exact same flow. So I broke it up and changed the order of things a bit.